docs/content/operations/rule-configuration.md

```diff
@@ -129,6 +129,22 @@ Period drop rules are of the form:
 
 The interval of a segment will be compared against the specified period. The period is from some time in the past to the current time. The rule matches if the period contains the interval.
 
+### Period Drop Before Rule
+
+Period drop before rules are of the form:
+
+```json
+{
+  "type" : "dropBeforeByPeriod",
+  "period" : "P1M"
+}
+```
+
+* `type` - this should always be "dropBeforeByPeriod"
+* `period` - A JSON Object representing ISO-8601 Periods
+
+The interval of a segment will be compared against the specified period. The period is from some time in the past to the current time. The rule matches if the interval before the period. If you just want to retain recent data, you can use this rule to drop the old data that before a specified period.
```

Comment:
Just so I'm clear, this is an alternative to doing a load by period followed by a drop forever?

Is this intended to be used in cases where the default load rule is generally used, but you want to configure the maximum age on a per-datasource basis?  If so can that be added to the docs?

Reply:
I get the situation now (from the issue description). Form this description it is not clear what scenarios it is desired to use this rule, can either a link to the issue or more detail on when this is appropriate be added here?

Reply:
> If you just want to retain recent data, you can use this rule to drop the old data that before a specified period.

Hmm @drcrallen do you think the quote is not enough to explain the desired use case of this rule? Would you please elaborate more on what details you want? Maybe it's better to add like `this rule is effectively dropForever + loadByPeriod (includeFuture = true)`?

Reply:
Maybe like below
```  
If you just want to retain recent data, you can use this rule to drop the old data that 
before a specified period and add a loadForever rule to follow it. 
Notes, dropBeforeByPeriod + loadForever is equivalent to dropForever + loadByPeriod (includeFuture = true)
```
?

Reply:
Sounds good to me.